sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -138,6 +139,47 @@ public Poller(Duration pollInterval, Function<PollResponse<T>, Mono<PollResponse
         this.autoPollingEnabled = true;
     }
 
+    /**
+     * Create a {@link Poller} instance with poll interval and poll operation. The polling starts immediately by invoking {@code pollOperation}.
+     * The next poll cycle will be defined by {@code retryAfter} value in {@link PollResponse}.
+     * In absence of {@code retryAfter}, the {@link Poller} will use {@code pollInterval}.
+     *
+     * <p><strong>Code Sample - Create poller object</strong></p>
+     * {@codesnippet com.azure.core.util.polling.poller.initialize.interval.polloperation}
```

Comment:
Need a new code sample

Reply:
yeah, just want to make sure, API surface change is good to go, before putting effort for Tests + Samples.